src/components/split-button/test/split-button.e2e.ts

```diff
@@ -0,0 +1,195 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
```

Comment:
Is there any particular reason for why these test files have been places in a sub-folder called `test`? We haven't done that anywhere else in this repo as far as I know, and I prefer not to separate the tests from the code being tested like that 🙂

Reply:
> Is there any particular reason for why these test files have been places in a sub-folder called test?

I think you are the one who did it. Maybe can you answer it 😆 ? You created the test file running the command `npm run generate`

Reply:
Never trust robots! 😅